spec/latest/core/core.jsonld

```diff
@@ -340,7 +340,7 @@
       "@type": "hydra:Link",
       "label": "collection",
       "comment": "Collections somehow related to this resource.",
-      "domain": "hydra:Resource",
+      "domain": "rdfs:Resource",

```

Comment:
I think this one should stay. If it's an target of a link then it would have to be a dereferencable resource, right?

Reply:
It's a domain. I think a link could be placed between a resource that may be not dereferencable and another resource that is actually dereferencable, i.e. inlined collection member, which may not be dereferencable and still may point to something that can be called.